tests/nativeapp/test_sf_sql_facade.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2024 Snowflake Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import mock
+
+from snowflake.cli._plugins.nativeapp.sf_facade import get_snowflake_facade
+from snowflake.cli.api.entities.common import get_sql_executor
+
+from tests.nativeapp.utils import (
+    SQL_EXECUTOR_EXECUTE,
+    SQL_EXECUTOR_EXECUTE_QUERIES,
+    mock_execute_helper,
+)
+
+# TODO: Add more test coverage + define contract
+# TODO: Add test for non-identifier names
```

Comment:
fast-follow PRs? What's the plan here?

Reply:
Yes, see note on my PR description. This class is not used in any code path (since Frank has removed apply_package_script in the meantime). So more tests and integ tests to be added in the upcoming PR for post deploy hooks.

Reply:
We should be able to cover the facade in isolation though...

Reply:
added tests